vowpalwabbit/core/src/reductions/epsilon_decay.cc

```diff
@@ -26,6 +28,126 @@ namespace epsilon_decay
 {
 float decayed_epsilon(uint64_t update_count) { return static_cast<float>(std::pow(update_count + 1, -1.f / 3.f)); }
 
+void epsilon_decay_data::update_weights(VW::LEARNER::multi_learner& base, VW::multi_ex& examples)
+{
+  auto model_count = static_cast<int64_t>(_scored_configs.size());
+  CB::cb_class logged{};
+  uint64_t labelled_action = 0;
```

Comment:
I assume label 0 and default `cb_class` object is desired behavior when the `find_if()` call fails?

Reply:
Updated so no updating happens if a label isnt found